human_general_analysis/nbconverted/Viz_simulated_experiments.py

```diff
@@ -155,29 +155,53 @@ def make_volcano_plot_simulated(
         # Plot
         colors = ["lightgrey", "#2c7fb8"]
         
-        f = sns.scatterplot(
-           data=simulated_DE_stats_df,
-            x="log2FoldChange",
-            y="padj_log10",
-            hue="gene group",
-            hue_order=["none", "DEG"],
-            style="gene group",
-            markers={
-                "none": ".",
-                "DEG": "o",
-            },
-            palette=colors,
-            linewidth=0,
-            alpha=0.5,
-            legend=False,
-            ax=axes[i],
+        if i == 0:
```

Comment:
I think these changes were also from the previous PR (merging in master should fix).

Reply:
See previous comment